utils/bundle_array.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2019 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+
+#ifndef UTILS_BUNDLE_ARRAY_H_
+#define UTILS_BUNDLE_ARRAY_H_
+
+#include "common/model/transfer.h"
+#include "utarray.h"
+
+/**
+ * @file bundle_array.h
+ * @brief Implementation of bundle array object
+ */
+
+typedef UT_array bundle_array_t;
+
+static UT_icd bundle_array_icd = {sizeof(bundle_transactions_t), 0, 0, 0};
+
+/**
+ * Allocate memory of bundle_array_t
+ */
+void bundle_array_new(bundle_array_t** const bundle_array) { utarray_new(*bundle_array, &bundle_array_icd); }
+
+/**
+ * Free memory of bundle_array_t
+ */
+void bundle_array_free(bundle_array_t** const bundle_array) {
```

Comment:
Alternatively, declare both `bundle_array_new` and `bundle_array_free` as `static inline` functions.